main/main.go

```diff
@@ -202,8 +201,8 @@ func main() {
 
 	// Each Listener goroutine will have an atomic bool that it can set to false to indicate an unrecoverable error
 	api.HandleHealthCheck(relayerHealth)
```

Comment:
Can we pass the logger to the health check handler as well, and emit a fatal log when triggered? I think those logs would be useful to have directly from the application, and it would make that handler interface consistent with the other two.